certified-connectors/CloudConvert/readme.md

```diff
@@ -0,0 +1,24 @@
+# CloudConvert Connector
+
+[CloudConvert](https://cloudconvert.com) is a file converter service - more than 200 different audio, video, document, ebook, archive, image,
+spreadsheet and presentation formats supported. The connector allows to convert files, compress files and create website
+screenshots using the CloudConvert service.
+
```

Comment:
@josiasmontag please review the readme.md file, all the required headings should be present. You can refer to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1).